src/util/server.c

```diff
@@ -427,6 +427,21 @@ errno_t server_common_rotate_logs(struct confdb_ctx *confdb,
     return EOK;
 }
 
+errno_t server_common_get_debug_level(struct confdb_ctx *confdb,
```

Comment:
confdb and conf_path are unused. I think you can just return `debug_level` instead of `errno_t` since `SSSDBG_INVALID` is still current value of the debug level and it can be reported as such. However, it is set only during process initialization so it can really be obtained by callers.

Reply:
Done